CHANGELOG.txt

```diff
@@ -2,6 +2,13 @@
 CHANGELOG
 =========
 
+-------------------------------------------------------------------------------
+April 13, 2021 1.0.1
+-------------------------------------------------------------------------------
+
+- Added Area under the Curve (AUC) metric to Recommenders.
+- Dropped installation using wheel
```

Comment:
why to drop installation via wheel?

Reply:
Consistent with what we now have in other libraries and does not really buy anything if you can do `pip install`.